kubetest/gke.go

```diff
@@ -50,6 +51,8 @@ var (
 	// m[2]: pool name (passed to e2es)
 	// m[3]: unique hash (used as nonce for firewall rules)
 	poolRe = regexp.MustCompile(`zones/([^/]+)/instanceGroupManagers/(gke-.*-([0-9a-f]{8})-grp)$`)
+
+	urlRe = regexp.MustCompile(`https://.*/`)
```

Comment:
This is a typical pattern for `MustCompile`, since you'd rather panic earlier rather than later. You can't `const` it in the function because it's not const, and it's a waste to do it every function call.